URBNSwiftyConvenience/Classes/UIView+Additions.swift

```diff
@@ -8,20 +8,53 @@
 
 import UIKit
 
+public struct SafeAreaEdges: OptionSet {
```

Comment:
In the recent past we have decided against `OptionSet`'s for very similar use cases. Why use it here.

Reply:
Why did we decide against them?